build.gradle.kts

```diff
@@ -48,13 +48,12 @@ tasks.withType<Test> {
 }
 
 tasks.named<Test>("test") {
-    include("**/*Test.class")
-    exclude("**/*Spec.class")
+    include("no/nav/syfo/**")
```

Comment:
Jeg har endret på konfigurasjonen slik at disse slowTestene kan ligge i den mappen de naturlig hører hjemme. F.eks en repository test skal ligge i repository mappen og ikke i en slowtest mappe. For å få dette til å kjøre må slowTests filene slutte på *Spec for å kjøre.

Reply:
Ok, da endrer jeg de tilbake slik du vil ha de.